src/rubrix/server/datasets/api.py

```diff
@@ -183,3 +182,38 @@ def open_dataset(
 
     """
     service.open_dataset(name, owner=current_user.current_group)
+
+
+@router.put(
+    "/{name}:copy",
+    operation_id="copy_dataset",
+    response_model=Dataset,
+    response_model_exclude_none=True,
+)
+def open_dataset(
+    name: str,
+    copy_request: CopyDatasetRequest,
+    service: DatasetsService = Depends(create_dataset_service),
+    current_user: User = Security(auth.get_user, scopes=[]),
+) -> Dataset:
+    """
+    Closes a dataset. This operation will releases backend resources
```

Comment:
Is this the right doc string? If so, maybe the method should be called `close_dataset`?

Reply:
yes, done